msg/JogJoint.msg

```diff
@@ -0,0 +1,11 @@
+ # Used in time-stamping the message.
+ Header header
+ 
+ # Name list of the joints. You don't need to specify all joints of the
+ # robot. Joint names are case-sensitive.
+ string[] joint_names
+ 
+ # Relative displacement of the joints to jog. The order must be
```

Comment:
"Relative displacement of the joints to jog" --> "Relative displacement of the joints listed in `joint_names`, the order must be identical."